lib/jsbeautifier.js

```diff
@@ -163,7 +163,7 @@ JsBeautifierTask.prototype.run = function() {
     }
 
     var sourceFiles = this.task.files;
-    if (sourceFiles && sourceFiles.length > 0) {
+    if (sourceFiles && sourceFiles.length > 0 && sourceFiles[0].src.length > 0) {
```

Comment:
This new condition will check for src on only first file, what if file other than 0th index is missing.